x-pack/test/api_integration/apis/slos/fixtures/create_slo.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { CreateSLOInput } from '@kbn/slo-schema';
+
+export const sloData: CreateSLOInput = {
+  name: 'Test SLO for api integration',
+  description: 'Fixture for api integration tests',
+  indicator: {
+    type: 'sli.kql.custom',
+    params: {
+      index: 'kbn-data-forge*',
+      filter: 'system.network.name: eth1',
```

Comment:
Since we generate data for the past 16min, we could add a `@timestamp > now-17m` to speed up the initial queries (no need to backfill empty data for the past 7 days)